engine/glfw/js/library_glfw.js

```diff
@@ -578,9 +578,11 @@ var LibraryGLFW = {
                   //In case when user change gamepad while browser in background (minimized)
                   GLFW.disconnectJoystick(joy);
                 }
+                var generic_id = (gamepad.mapping == "standard") ? "Standard Gamepad" : gamepad.id;
```

Comment:
Anything with mapping set to "standard" will confirm to the w3c standard gamepad definition in terms of buttons, axis and general layout.

Reply:
Why not set the gamepad.id to "Standard Gamepad" ?

* Don't we want to use the "Standard Gamepad" in these cases?
* The users most likely _don't_ want to use potentially different mappings of "Brandname 1" and "Brandname 2" if it's known to be a standard mapping?

Reply:
Yeah, maybe you're right! I'll change this and default to "Standard Gamepad"